lib/puppet/provider/package/brewcask.rb

```diff
@@ -45,15 +45,20 @@ def query
   end
 
   def install
+    install_cmd = ['brew']
     if install_options.any?
-      execute ["brew", "install", "Caskroom/cask/#{resource[:name]}", *install_options].flatten, command_opts
```

Comment:
![](https://i.giphy.com/AjYsTtVxEEBPO.gif)

I was only looking at why Canary wasn't installing for me last week and completely overlooked this.